tests/fingerprints/physiochemical_properties.py

```diff
@@ -0,0 +1,100 @@
+import numpy as np
+from scipy.sparse import csr_array
+
+from skfp.fingerprints import PhysiochemicalPropertiesFingerprint
+
+
+def test_physiochemical_properties_bp_bit_fingerprint(smiles_list):
+    pp_fp = PhysiochemicalPropertiesFingerprint(
+        variant="BP", sparse=False, count=False, n_jobs=-1
+    )
+    X_skfp = pp_fp.transform(smiles_list)
+
+    assert isinstance(X_skfp, np.ndarray)
```

Comment:
Many security tools won't allow for the use of `assert` and instead they'll insist on using 
`if not condition: raise AssertionError ("[rror message")`

Reply:
This does not apply to tests